javaslang/src/main/java/javaslang/collection/Iterator.java

```diff
@@ -1920,27 +1933,31 @@ public T getNext() {
             final Iterator<T> that = this;
             return new AbstractIterator<T>() {
 
-                private T next = null;
+                private T next;
```

Comment:
Will check it. had problems with `(1,1,2,2,1,1).span(i -> i == 1)`. unordered and sorted collections tests failed...
So I used `(1,1,2,2,3,3).span(i -> i % 2 == 1)` instead.
These tests should fail with unmodified Iterator class!